src/main/java/edu/cornell/kfs/module/ar/batch/service/impl/CuContractsGrantsInvoiceCreateDocumentServiceImpl.java

```diff
@@ -0,0 +1,69 @@
+package edu.cornell.kfs.module.ar.batch.service.impl;
+
+import org.kuali.kfs.integration.cg.ContractsAndGrantsBillingAward;
+import org.kuali.kfs.integration.cg.ContractsAndGrantsBillingAwardAccount;
+import org.kuali.kfs.krad.util.ObjectUtils;
+import org.kuali.kfs.module.ar.businessobject.ContractsGrantsInvoiceDocumentErrorLog;
+import org.kuali.kfs.module.ar.businessobject.ContractsGrantsInvoiceDocumentErrorMessage;
+import org.kuali.kfs.module.ar.service.impl.ContractsGrantsInvoiceCreateDocumentServiceImpl;
+import org.kuali.kfs.sys.businessobject.SystemOptions;
+import org.kuali.rice.core.api.util.type.KualiDecimal;
+import org.springframework.util.CollectionUtils;
+
+import java.sql.Date;
+import java.util.Collection;
+import java.util.List;
+import java.util.Map;
+
+public class CuContractsGrantsInvoiceCreateDocumentServiceImpl extends ContractsGrantsInvoiceCreateDocumentServiceImpl {
+
+    @Override
+    protected void storeValidationErrors(Map<ContractsAndGrantsBillingAward, List<String>> invalidGroup, Collection<ContractsGrantsInvoiceDocumentErrorLog> contractsGrantsInvoiceDocumentErrorLogs, String creationProcessTypeCode) {
```

Comment:
I think it would be helpful for this method have a Javadoc comment that explains what was changed, given that this override appears to be almost identical to the original except for a few lines.